scripts/multi_component_hydrogen_bond_propensity/multi_component_hydrogen_bond_propensity_report.py

```diff
@@ -340,6 +343,11 @@ def main(structure, work_directory, failure_directory, library, csdrefcode):
     # for each coformer in the library, make a pair file for the api/coformer and run a HBP calculation
     for i, f in enumerate(coformer_files):
         molecule_file, coformer_name = make_pair_file(api_molecule, tempdir, f, i + 1)
+        if not io.CrystalReader(f)[0].molecule.is_3d:
+            failure_warning = f"Could not run for {coformer_name} no 3d coordinates present."
+            failures.append(failure_warning)
+            warnings.warn(failure_warning)
+            continue
```

Comment:
Not too sure about this one. We sometimes use the scripts on 2D structures when a crystal structure is not available. I see that it still runs but maybe we can have a flag to skip the warning and appending it to the error list. Thoughts?